builder/core/data.py

```diff
@@ -132,7 +132,7 @@ class PKG_TOOLS(Enum):
         'pkg_update': 'apt-get -qq update -y',
         'pkg_install': 'apt-get -qq install -y',
         'variables': {
-            'python': "python3.8",
+            'python': "python3",
```

Comment:
Just explaining why this explicitly said "python3.8"...
This was set 2 years ago, in this PR: https://github.com/awslabs/aws-crt-builder/pull/198

back then, on some of our docker images, installing "python3" would get you python 3.6, but you could get something newer by specifying "python3.8"

We've probably stopped using those super old images, so this makes sense.
But do a census to be sure...

There are still a few other places we specify python version, if you want to do a search for "python3."